conans/client/graph/compute_pid.py

```diff
@@ -18,6 +19,7 @@ def compute_package_id(node, new_config):
     embed_mode = new_config.get("core.package_id:default_embed_mode", default="full_mode")
     python_mode = new_config.get("core.package_id:default_python_mode", default="minor_mode")
     build_mode = new_config.get("core.package_id:default_build_mode", default=None)
+    conf_names = new_config.get("core.package_id:confs", default=[], check_type=list)
```

Comment:
Not sure if we want this to be ``core``, or it would be a ``profile`` one, something that users would prefer to define in the ``profiles``?

Reply:
Yes, that was what I was trying to figure out. I opened the draft to see how it looked like, and, definitely, it should go through the user profile 😉